### ISSUE TITLE
Finetuning on LIBERO-90

### ISSUE THREAD

**Issue by rohan-bansal** [WEIRDLabUW/unified-world-model#20]
Hello, this is a great work.

Currently, if I just evaluate the LIBERO-90 checkpoint on the LIBERO-90 dataset, it achieves around a 22-24% success rate (20 rollouts for each env). This makes sense, since it is a pretraining checkpoint. 

I have also finetuned for 20k steps, and the success rate remains about the same. In order to properly evaluate on LIBERO-90 by itself, should I finetune for many more steps? Or should I weight the action loss such that it is not equal to the video prediction loss?

**Comment by zchuning** [comment]
How did you finetune on LIBERO-90? Did you finetune on the combined dataset or single-task dataset? UWM is not a multi-task policy. It needs to be finetuned on each individual downstream task.

**Comment by zchuning** [comment]
Closing due to inactivity. Feel free to reopen for follow-up questions.
